utils/preflight_check.py

```diff
@@ -433,30 +433,30 @@ def main(
     parameters = get_taskgraph_parameters()
     run_taskgraph(parsed_args.config, get_taskgraph_parameters())
 
-    match Choices[parsed_args.only] if parsed_args.only else None:
-        case Choices.task_group:
-            pretty_print_task_graph()
-        case Choices.artifacts:
-            pretty_print_artifacts_dir()
-        case Choices.training_config:
-            pretty_print_training_config(get_taskgraph_parameters())
-        case Choices.graph:
-            serve_taskgraph_file(
-                parsed_args.graph_url,
-                parsed_args.open_graph,
-                parsed_args.persist_graph,
-                open_in_browser,
-            )
-        case None:
-            pretty_print_task_graph()
-            pretty_print_artifacts_dir()
-            pretty_print_training_config(parameters)
-            serve_taskgraph_file(
-                parsed_args.graph_url,
-                parsed_args.open_graph,
-                parsed_args.persist_graph,
-                open_in_browser,
-            )
+    choice = Choices[parsed_args.only] if parsed_args.only else None
+    if choice == Choices.task_group:
+        pretty_print_task_graph()
+    elif choice == Choices.artifacts:
+        pretty_print_artifacts_dir()
+    elif choice == Choices.training_config:
+        pretty_print_training_config(get_taskgraph_parameters())
+    elif choice == Choices.graph:
+        serve_taskgraph_file(
+            parsed_args.graph_url,
+            parsed_args.open_graph,
+            parsed_args.persist_graph,
+            open_in_browser,
+        )
+    elif choice is None:
```

Comment:
These changes look completely unrelated and should be broken out into a separate PR. I don't see any changes here.